runtime/lua/vim/lsp.lua

```diff
@@ -1842,10 +1842,16 @@ function lsp.get_active_clients(filter)
 
   local t = filter.bufnr and (all_buffer_active_clients[resolve_bufnr(filter.bufnr)] or {})
     or active_clients
+
+  if #active_clients == 0 then
```

Comment:
`active_clients` are indexed by client id, right? So it can have holes. If client with id 1 is stopped, than this condition returns too early.

Reply:
active_clients is table the value is client id